src/redis-cli.c

```diff
@@ -9108,6 +9108,19 @@ static void findBigKeys(int memkeys, unsigned memkeys_samples) {
     printf("\n# Scanning the entire keyspace to find biggest keys as well as\n");
     printf("# average sizes per key type.  You can use -i 0.1 to sleep 0.1 sec\n");
     printf("# per 100 SCAN commands (not usually needed).\n\n");
+    
+    /* Use readonly in cluster */
+    if(config.cluster_mode){
+        read_reply= redisCommand(context,"READONLY");
+        if(read_reply == NULL){
+            fprintf(stderr, "\nI/O error\n");
+            exit(1);
+        }else if (read_reply->type == REDIS_REPLY_ERROR) {
```

Comment:
Whitespace suggestion redis style.

```suggestion
    if (config.cluster_mode) {
        read_reply = redisCommand(context, "READONLY");
        if (read_reply == NULL){
            fprintf(stderr, "\nI/O error\n");
            exit(1);
        } else if (read_reply->type == REDIS_REPLY_ERROR) {
```

Reply:
thanks for you advice